ruby/rspec/chrome_example_spec.rb

```diff
@@ -24,9 +24,14 @@
   before(:each) do |test|
     caps = {
         browser_name: 'chrome',
-        platform_version: 'Windows 10',
+        # platform_version now becomes platform_name with the W3C bridge.
+        # For more details see: https://www.rubydoc.info/gems/selenium-webdriver/Selenium/WebDriver/Remote/W3C/Capabilities
+        platform_name: 'windows 10',
         browser_version: '61.0',
-        "goog:chromeOptions": {w3c: true},
+        # w3c is note enabled by default in Chrome, set the w3c setting to true
+        "goog:chromeOptions" => {w3c: true},
+        # use sauce:options to handle all saucelabs.com-specific capabilities such as:
+        # username, accesskey, build number, test name, timeouts etc.
         "sauce:options" => {
             name: test.full_description,
             seleniumVersion: '3.11.0',
```

Comment:
I tested with the latest version of Selenium (3.141.59), and it seems to work. I think we should try to reflect that version in all these examples.

Reply:
Updated Selenium Version to 3.141.59 and tested. Still runs!